src/notifications/tests/test_tasks.py

```diff
@@ -41,35 +40,35 @@ def test_get_notification_emails(self):
         # No notifications to start
```

Comment:
This comment doesn't match the assert... need to update this that the owner has turned on by default so expecting 1

Reply:
fixed